roles/ipaserver/README.md

```diff
@@ -359,6 +359,7 @@ Special Variables
 Variable | Description | Required
 -------- | ----------- | --------
 `ipaserver_install_packages` | The bool value defines if the needed packages are installed on the node. (bool, default: true) | no
+`ipaserver_install_packages_only` | The bool value defines if the deployment will continue after the needed packages are installed on the node. (bool, default: true) | no
```

Comment:
`ipaserver/defaults/main.yml` has `ipaserver_install_packages_only: false`, shouldn't the default be `false`?

Also, I think the value should be true if only the packages should be installed, so, in my opinion, description should be something like "When set to true, required packages will be installed, but server will not be configured. (bool, default: false)"